### PR TITLE
Fix pages layout

### PR DESCRIPTION
**Pull request by kotesh-arya**

### CODE REVIEW

File: Components/LogIn/login.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h1 class="header-large">LOG IN</h1>
         </div>
         <button class="btn btn-primary login-btn"> <a class="home-page-link"
                 href="/Components/Homepage/homepage.html">LOG IN</a></button>
-        <h2><a class="signup-link" href="/Components/SignUp/signup.html">Create new account &gt;</a></h2>
```
Comment (vinc3nati) [inline Components/LogIn/login.html:45]: nice use of html entity code

**Review by vinc3nati — APPROVED** [review]

File: Components/SignUp/signup.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
     text-align: left;
     width: 24rem;
     text-align: center;
```
Comment (zabihshaik) [inline Components/SignUp/signup.css:3]: text-align is repeated

**Review by zabihshaik — APPROVED** [review]
Code Structure looks Good.

File: Components/SignUp/signup.css (hunk 2 above)
Comment (kotesh-arya) [reply Components/SignUp/signup.css:3]: Thanks for pointing out, I have updated that 👍🏻

**Pull request merged**